kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/auth/AuthLocalData.java

```diff
@@ -120,15 +122,21 @@ public String getUserID() {
 		return signInSharedPreferences.getString(USER_ID_KEY, null);
 	}
 
+	@Override
+	public String getEcosystemUserID() {
+		return signInSharedPreferences.getString(ECOSYSTEM_USER_ID_KEY, null);
+	}
+
 	@Override
 	public AuthToken getAuthTokenSync() {
 		String token = signInSharedPreferences.getString(TOKEN_KEY, null);
 		String appID = signInSharedPreferences.getString(APP_ID_KEY, null);
 		String userID = signInSharedPreferences.getString(USER_ID_KEY, null);
+		String ecosystemUserID = signInSharedPreferences.getString(ECOSYSTEM_USER_ID_KEY, null);
```

Comment:
use getEcosystemUserID()

Reply:
just keeping the same for every param... 
we can change but doesn't really matter

Reply:
just duplicate code - but not critical